osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/GetActualPowerQualityCommandExecutorTest.java

```diff
@@ -298,39 +297,31 @@ private CosemObject createObject(
       final String tag,
       final String scalerUnitValue,
       final MeterType meterType) {
-    final CosemObject object = new CosemObject();
-    object.setClassId(classId);
-    object.setObis(obis);
-    object.setTag(tag);
-    if (scalerUnitValue != null) {
-      object.setAttributes(this.createScalerUnitAttributeList(scalerUnitValue));
-    }
-    object.setMeterTypes(this.getMeterTypes(meterType));
-
-    return object;
+    return this.createCosemObject(tag, classId, obis, scalerUnitValue, meterType);
   }
 
   private List<Attribute> createScalerUnitAttributeList(final String value) {
-    final Attribute scalerUnitAttribute = new Attribute();
-    scalerUnitAttribute.setId(ATTRIBUTE_ID_SCALER_UNIT);
-    scalerUnitAttribute.setValue(value);
-    return Collections.singletonList(scalerUnitAttribute);
+    return List.of(
+        new Attribute(
+            ATTRIBUTE_ID_SCALER_UNIT,
+            "descr",
+            null,
+            DlmsDataType.DONT_CARE,
```

Comment:
What is a DONT_CARE?

Reply:
In this test the DlmsDataType doesn't matter. We could choose any value here, but DONT_CARE makes it clear it doesn't matter.